pinax/stripe/tests/test_actions.py

```diff
@@ -2986,6 +2986,13 @@ def test_sync_not_custom_account(self):
         self.assertNotEqual(account.type, "custom")
         self.assert_common_attributes(account)
 
+    def test_deauthorize_account(self):
```

Comment:
Do we need a mock here to prevent hitting the Stripe API?

Reply:
If stripe API was hit you would see an error. I just ran the test suite offline and all tests are passing.